eng/Versions.props

```diff
@@ -300,6 +300,7 @@
       https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
       https://dotnet.myget.org/F/dotnet-corefxtestdata/api/v3/index.json;
       https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json;
+      https://dotnet.myget.org/F/experimental-vs-packages/api/v3/index.json;
```

Comment:
@dibarbet Which packages were in this feed? They probably should have been uploaded to roslyn-tools.

Reply:
It was a new version of Microsoft.VisualStudio.Shell.Framework that isn't published here yet - https://www.nuget.org/packages/Microsoft.VisualStudio.Shell.Framework/ .  It has the new partial load stuff in it that we need to test with.

Should that be in roslyn tools?